GlobalViolence/R/DataPrep_Graduate.R

```diff
@@ -186,16 +186,20 @@ variants <- c("low","mid","upp")
 # you can change this:
 # mc.cores = (detectCores() - 1)
 # to use fewer cores. It'll work even if mc.cores is 1 or 2.
+# Actually this strategy did not work for me; I just kept your previous strategy
+# I used 8 here because I could at work. But don´t think it is duable usually
+
 
 for (i in 1:length(variants)){
   GBDi <- readRDS(file.path(gbd.folder, paste0("GBD",variants[i],".rds"))) 
     GBDi %>% split(list(GBDi$location,GBDi$sex,GBDi$year), drop = TRUE) %>% 
-	  mclapply(GBD.chunk, mc.cores = (detectCores() - 1)) %>% 
+	  mclapply(GBD.chunk, mc.cores = 8) %>% 
```

Comment:
let's change this line back to what it was, ought to work w both packages loaded.

Reply:
also FYI you can just make the change, push and the PR updates to reflect this